### DISCUSSION TITLE
Welsh language translation

### DISCUSSION THREAD

**Discussion by edbennett** [pretix/pretix#3532] [Ideas] [answered]
This afternoon, [Tocyn Cymru](https://tocyn.cymru), a bilingual ticketing service used commonly in the Welsh public sector, has announced they will be shutting down in the next few weeks. They don't suggest an alternative service to migrate to, and there are no obvious services that have Welsh-language support (which is a requirement for public bodies in Wales).

Could Welsh please be added to the set of languages enabled on `translate.pretix.eu`, so that effort can be put towards making Pretix a viable alternative?

**Comment by raphaelm** [comment]
Yes, well try to add it in the next few days

**Comment by raphaelm** [comment, marked as answer]
Added it for the core system: https://translate.pretix.eu/projects/pretix/pretix/cy/

Happy to add the language to all plugins as well if it gains traction with translators :)
